### ISSUE TITLE
error in git clone https://github.com/ferrygun/RN2.git 

### ISSUE THREAD

**Issue by babyhockey** [ferrygun/PDFTableExtract#1]
Hi, it seems https://github.com/ferrygun/RN2.git does not exist anymore? I ran into the following errors:

Cloning into 'RN2'...
fatal: could not read Username for 'https://github.com': No such device or address

Could you help fix this? BTW, excellent work in PDFTableExtract!! Thank you so much!

**Comment by 519994824** [comment]
I met the same question. Have you fixed it? Now i‘m turning to use Pytorch...

**Comment by dreamflown** [comment]
obviously, there is no repostry at  https://github.com/ferrygun which named RN2

**Comment by RitwickJoshi** [comment]
repo name is changed keras-retinanet

**Comment by CaptainStabs** [comment]
> repo name is changed keras-retinanet

I'm not sure that this is true.

![image](https://user-images.githubusercontent.com/40151222/116697481-39341a00-a991-11eb-85ea-61fbba45239b.png)

RN2 would be a separate folder within keras-retinanet (the output is what it is supposed to be, not what it actually is)
